app/controllers/hmis_api/graphql_controller.rb

```diff
@@ -0,0 +1,55 @@
+module HmisApi
+  class GraphqlController < HmisApi::BaseController
+    # If accessing from outside this domain, nullify the session
+    # This allows for outside API access while preventing CSRF attacks,
+    # but you'll have to authenticate your user separately
+    # protect_from_forgery with: :null_session
+
+    def execute
```

Comment:
Let's add a `before_action` or some other method to protect any response behind `ENV['ENABLE_HMIS_API'] == 'true'`

Reply:
Do we want this to go into `HmisApi::BaseController`? And is it necessary? These controllers are all in routes that are behind `ENV['ENABLE_HMIS_API'] == 'true`